modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -786,4 +783,11 @@ def runPublishingE2ETests(appCommitishName, testBranch, repo) {
   )
 }
 
+def setFullCommitHashEnvar() {
```

Comment:
I think rather than have this set an env var it should just return the full commit hash as a string. 

Setting the env var effectively just sets a global variable and we don't need that really.